misk-redis/src/main/kotlin/misk/redis/RealRedis.kt

```diff
@@ -1,10 +1,16 @@
 package misk.redis
 
+import arrow.core.Either
```

Comment:
Should be adding the arrow library for this type?

Reply:
+1, would much rather use Kotlin standard library vs. pulling in an opinionated library for a simple type.